spec/system/bank_holidays_spec.rb

```diff
@@ -270,8 +270,8 @@
     end
   end
 
-  context "GA4 tracking" do
-    it "has GA4 tracking on the .ics file links" do
+  context "with GA4 tracking" do
+    it "on the .ics file links" do
```

Comment:
This context and test name read strangely together. "with GA4 tracking it on the .ics file links" doesn't make sense.

Perhaps they should be reversed? 
i.e.

```ruby
describe ".ics file links" do
 it "has GA4 tracking" do
```

Reply:
Done